cmd/certsuite/run/run.go

```diff
@@ -39,8 +39,8 @@ func NewCommand() *cobra.Command {
 	runCmd.PersistentFlags().Bool("include-web-files", false, "Save web files in the configured output folder")
 	runCmd.PersistentFlags().Bool("enable-data-collection", false, "Allow sending test results to an external data collector")
 	runCmd.PersistentFlags().Bool("create-xml-junit-file", false, "Create a JUnit file with the test results")
-	runCmd.PersistentFlags().String("tnf-image-repository", "quay.io/redhat-best-practices-for-k8s", "The repository where TNF images are stored")
-	runCmd.PersistentFlags().String("tnf-debug-image", "certsuite-probe:v0.0.7", "Name of the certsuite-probe image")
+	runCmd.PersistentFlags().String("certsuite-image-repository", "quay.io/redhat-best-practices-for-k8s", "The repository where Certsuite images are stored")
+	runCmd.PersistentFlags().String("certsuite-debug-image", "certsuite-probe:v0.0.7", "Name of the certsuite-probe image")
```

Comment:
`certsuite-debug-image` --> `certsuite-probe-image-repository`
This is non-backward compatible change. @sebrandon1 , probably DCI, and QE will need to update to use these new flag names. Also the next certsuite release notes should reflect this changes.

Reply:
Ah true.  Yes this would be breaking for both DCI and QE.

On the flip side, does this variable need to be configurable at all?  I would argue we would want to just hardcode this variable to whichever debug image we want.

Reply:
We could potentially use this PR to remove this variable and just leave the hardcoded value?  We can discuss in scrum.

Reply:
We use that variable to refer to a debug image saved in a private repository, since we run tests on disconnected mode, so I'd say this variable is needed.

Reply:
Sorry, @bnshr, my previous comment was wrong, since this flag is just for images, not for repos.

Anyway, if I'm not (again) wrong, these two flags are solely used to [compose the final image name](https://github.com/redhat-best-practices-for-k8s/certsuite/blob/b2fb84c54900020c5ead43af428e16cbe0f8dc90/pkg/provider/provider.go#L184) for the debug (now probe) pods. So I believe we should simplify this with just one flag that includes the whole `registry/repo:tag` thing, like:

```
runCmd.PersistentFlags().String("certsuite-probe-image", "quay.io/redhat-best-practices-for-k8s/certsuite-probe:v0.0.7", "Certsuite probe image")
```

This suggestion would be non-backward compatible also. What do you all think? Would this work in DCI for disconnected environments, @ramperher ?

Reply:
If the repo is just used for the probe image, then I suppose it's fine.